flink-streaming-connectors/flink-connector-cassandra/src/main/java/org/apache/flink/connectors/cassandra/streaming/ClusterConfigurator.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connectors.cassandra.streaming;
+
+import com.datastax.driver.core.Cluster;
+
+public interface ClusterConfigurator {
```

Comment:
It seems like "reinvent the wheel" build a Builder on top of Datastax API and I think it is not enough  to set hosts on Cluster.Builder, there are a lot of useful settings.